src/config.js

```diff
@@ -4,6 +4,8 @@ import winston from 'winston';
  * BOT CONFIGURATION:
  */
 
+export const HIPCHAT_CALLBACK_URL = process.env.HIPCHAT_CALLBACK_URL || '';
```

Comment:
Yea this can be added into the installation flow. Should I add another field on the admin settings page for this?